python/add_mapillary_tag_from_exif.py

```diff
@@ -51,7 +51,7 @@
     Use from command line as: python add_mapillary_tag_from_exif.py root_path [sequence_uuid]
     '''
 
-    Fetch authetication info from env
+    # Fetch authetication info from env
```

Comment:
you may as well correct the spelling error then: authetication -> authentication